tests/by-util/test_timeout.rs

```diff
@@ -143,18 +143,29 @@ fn test_kill_after_long() {
         .no_stderr();
 }
 
+/// Test that a command that is executed in a sub-shell is killed/terminated.
 #[test]
-fn test_kill_subprocess() {
+fn test_kill_subshell() {
     new_ucmd!()
         .args(&[
-            // Make sure the CI can spawn the subprocess.
-            "10",
+            "1",
+            // Spawn command in sub-shell.
             "sh",
             "-c",
-            "sh -c \"trap 'echo xyz' TERM; sleep 30\"",
+            // Set a trap to print 'xyz' when the sub-shell receives a
+            // TERM signal.
+            "trap 'echo xyz' TERM; sleep infinity",
         ])
+        // We require failing only in order to check the exit code.
         .fails()
         .code_is(124)
-        .stdout_contains("xyz")
-        .stderr_contains("Terminated");
```

Comment:
For reference, I removed the check for stderr containing `Terminated`.

Reply:
That sounds good! I believe that "Terminated" depends on the shell or something like that. Definitely not something we should rely on! Why did you remove the "xyz" part?

Reply:
Yep, I agree!